### DISCUSSION TITLE
How to call the method of C # for the object variable defined by JavaScript

### DISCUSSION THREAD

**Discussion by WangDeBug** [sebastienros/jint#1013] [Q&A]
js code:
`var objstu=obj.create(param); 
objstu.study( );`

Jint error：
Property 'study' of object is not a function

I'm sure jint can be implemented, but I don't know how to write it

**Comment by lahma** [comment]
`objstudy.study = function() {};`

**Reply by WangDeBug** [reply]
I hope function 'study' is the method of c# and implements the c# method of JS operation

**Reply by WangDeBug** [reply]
`_engine.SetValue("obj"，new obj()) `

**Reply by lahma** [reply]
Please see all the interop tests in this repository, they describe very many ways to do things.
